Configuration/EventContent/python/EventContent_cff.py

```diff
@@ -642,6 +642,8 @@ def SwapKeepAndDrop(l):
                               outputCommands = FEVTDEBUGHLTEventContent.outputCommands+[
                                   'keep *_hltSiStripClusters2ApproxClusters_*_*'
                               ])
+phase2_muon.toModify(FEVTDEBUGHLTEventContent, 
```

Comment:
Do you need it here, under FEVTDEBUGHLT? 
I assume you need it under FEVTDEBUG, which is [few lines above](https://github.com/cms-sw/cmssw/blob/a0c7e4868d3f543c7dfc7ae46989166254c7e43d/Configuration/EventContent/python/EventContent_cff.py#L612-L626).

Do I misunderstand something?

Reply:
Technically, having it here is also OK as we always do FEVTDEBUGHLT as output, not FEVTDEBUG.

Reply:
Feedback? @waredjeb

Reply:
@srimanob sorry for the late reply. We just need this for debugging, FEVTDEBUGHLT is completely fine and we don't need it in FEVTDEBUG

Reply:
FEVTDEBUGHLT is an extension of FEVTDEBUG.